config/config-sample.js

```diff
@@ -1,5 +1,7 @@
 module.exports = {
-    base_url: 'http://127.0.0.1:3000',
+    port: 36902, // for production environment
+    dev_port: 3000, // for gulp watch
+    base_url: 'http://127.0.0.1:3000/',
```

Comment:
Something feels wrong here even if it's only an example - why does base_url has dev_port in it? Why two ports? How about: `{port: .. , base_url: .. , mode: "dev" or "prod"}`, where autoreload/watch is enabled if mode==dev (maybe logging settings could also change)